test/integration/testdata/valuevalidation/alpha.go

```diff
@@ -0,0 +1,72 @@
+package valuevalidation
+
+import (
+	"k8s.io/kube-openapi/pkg/common"
+	"k8s.io/kube-openapi/pkg/validation/spec"
+)
+
+// Dummy type to test the openapi-gen API rule checker.
+// The API rule violations are in format of:
+// -> +k8s:validation:[validation rule]=[value]
+
+// +k8s:validation:maxProperties=5
```

Comment:
Wait, when did we decide to use `k8s:validation:` prefix? That's not useful. it should be `+maxProperties`.

Reply:
removed the prefix, added a whitelist to prevent collision with other marker comments